swift/Samples/ModalContainer/Sources/ModalContainerScreen.swift

```diff
@@ -0,0 +1,50 @@
+import WorkflowUI
+
+
+/// A `ModalContainerScreen` displays a base screen and optionally one or more modals on top of it.
+public struct ModalContainerScreen: Screen {
+
+    /// The base screen to show underneath any modally presented screens.
+    public var baseScreen: AnyScreen
```

Comment:
Should we use generics here instead? cc: @bencochran

Reply:
+1, I vote yes if its possible.

Reply:
Would love to see this become `ModalContainerScreen<Base, Modal>` or similar.

Reply:
will do!